index.html

```diff
@@ -69,5 +70,225 @@ <h2 class="connect">Let's Connect</h2>
+          </div>
+          <div class="container container-flex">
+            <h2 class="work-title">
+              Multi-Post Stories
+              <br />
+              Gain+Glory
+            </h2>
+          </div>
+
+          <div class="work-tags">
+            <p class="tag1">Javascript</p>
+            <p class="tag1">HTML</p>
+            <p class="tag1">CSS</p>
+          </div>
+          <div class="btn">
+            <button type="button" class="btn-pop">
+              See Project
+            </button>
+          </div>
+        </article>
+
+
+
+        <article class="work-card">
+          <div class="container container-flex">
+            <img src="./asset/cat1.jpg" alt="Tonic work" class="work-image" />
+          </div>
+          <div class="container container-flex">
+            <h2 class="work-title">
+              Multi-Post Stories
+              <br />
+              Gain+Glory
+            </h2>
+          </div>
+
+          <div class="work-tags">
+            <p class="tag1">Javascript</p>
+            <p class="tag1">HTML</p>
+            <p class="tag1">CSS</p>
+          </div>
+          <div class="btn">
+            <button type="button" class="btn-pop">
+              See Project
+            </button>
+          </div>
+        </article>
+
+        <article class="work-card">
+          <div class="container container-flex">
+            <img src="./asset/cat2.jpeg" alt="Tonic work" class="work-image" />
+          </div>
+          <div class="container container-flex">
+            <h2 class="work-title">
+              Multi-Post Stories
+              <br />
+              Gain+Glory
+            </h2>
+          </div>
+
+          <div class="work-tags">
+            <p class="tag1">Javascript</p>
+            <p class="tag1">HTML</p>
+            <p class="tag1">CSS</p>
+          </div>
+          <div class="btn">
+            <button type="button" class="btn-pop">
+              See Project
+            </button>
+          </div>
+        </article>
+      </div>
+    </section>
+
+
+    <div class="aboutme">
+      <h2>
+        About Me
+      </h2>
+
+      <p>
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <div class="btn">
+      <button type="button" class="btn-pop" id="GetResume">
+        Get My Resume
+      </button>
+    </div>
+
+
+
+    <div class="">
+      <article class="work-card">
+        <div class="container container-flex">
+          <h2>Languages</h2>
+          <img src="asset/icon-languages.svg" alt="languages" />
+        </div>
+        <div>
+          <ul class="work-tags">
+            <li class="tag1">CSS</li>
+            <li class="tag1">Javascipt</li>
+            <li class="tag1">HTML</li>
+            <li class="tag1">Ruby</li>
+          </ul>
```

Comment:
[ Optional Changes ] Any change marked as optional won't stop your project from being approved but will help improve your codebase and make you become a better developer by fixing them. 

- [ ] The previous reviewers did not spot these changes but since I have eyes for design so much, please kindly make the badges look like the one given in the Figma design. 
- [ ] It would be nice if you add the See Project button to each card in your design. It's a requirement that you have it but the previous reviewers did not spot that. Please fix that before you start your next milestone next week. 🙏 
- [ ] Also kindly add margin-bottom space from each of the cards to match the Figma design

- **The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/164768376-6207de66-a474-4002-a661-7fb02737c65a.png)

- **Your design**
![image](https://user-images.githubusercontent.com/50941074/164768778-00ccf1d0-07dd-4209-8180-eb96b22a5e23.png)

Reply:
@AmaduKamara Will do it in few mins. 👍 :)